lib/generators/pghero/templates/blocker_sample_sessions.rb.tt

```diff
@@ -0,0 +1,31 @@
+class <%= migration_class_name %> < ActiveRecord::Migration<%= migration_version %>
+  def change
+    create_table :pghero_blocker_sample_sessions do |t|
```

Comment:
Would it be good to capture `application_name` here since we're back to one user?

Reply:
It's there; called 'source'. I Ieft the names consistent with Postgres monitoring views (except where we were changing the data type like user) so that the data is a very close mirror to documentation on pg_stat_activity. The connection module in pghero aliases 'source' to 'application_name'. Same data.